go.mod

```diff
@@ -1,277 +1,181 @@
-module github.com/puppetlabs/vault-plugin-secrets-oauthapp/v3
+module github.com/openbao/openbao-plugin-secrets-oauthapp/v3
 
-go 1.18
+go 1.23.1
+
+toolchain go1.24.0
 
 require (
 	github.com/coreos/go-oidc v2.2.1+incompatible
-	github.com/golangci/golangci-lint v1.50.1
-	github.com/hashicorp/go-hclog v1.4.0
+	github.com/golangci/golangci-lint v1.64.6
```

Comment:
It'd be great to move this to a tools `go.mod` later, but a problem for another time.